tests/js/spec/views/issueList/tagFilter.spec.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * @jest-environment jest-environment-jsdom-fifteen
```

Comment:
This is needed as we are using a Jest version lower than 25. Please see: https://testing-library.com/docs/react-testing-library/setup/#jest-24-or-lower-and-defaults

Reply:
We're using jest 26

Reply:
@billyvg  I meant here https://github.com/getsentry/jest-visual-snapshot/blob/b97665bc92fa2313ef79196615f9e621518eade4/packages/jest-environment/package.json#L22

Reply:
I will double-check it, but I believe this was the cause of an issue

Reply:
I updated the package to now use jest-environment-jsdom v26